packages/zowe-explorer/src/trees/dataset/DatasetTree.ts

```diff
@@ -1351,6 +1351,16 @@ export class DatasetTree extends ZoweTreeProvider<IZoweDatasetTreeNode> implemen
             ZoweLogger.warn(e);
             return;
         }
+        await this.filterTreeByPattern(node, profile, pattern, true);
+    }
+
+    public async filterTreeByPattern(
+        node: IZoweDatasetTreeNode,
+        profile: imperative.IProfileLoaded,
+        pattern: string,
+        addToHistory: boolean
```

Comment:
(nitpick) making `addToHistory` optional has the same effect and avoids having to pass `false` when the caller doesn't want to add the filter to history